### ISSUE TITLE
`Docker-compose.yml`: Add prefix to service and container name

### ISSUE THREAD

**Issue by luca-rigutti** [pieroit/cheshire-cat#163]
Hi,

If there is not problem with something else on the code, I suggest to add a prefix to service and container to avoid collision with other service used with docker.

An example of this can be:

```yaml
  cheshire-cat-core:
      build:
        context: ./core
      container_name: cheshire_cat_core
```

And also for the `admin` and `vector-memory`

Thanks

**Comment by pieroit** [comment]
Yes this is a good idea can you do it in a PR?

The only place I recall there is a direct reference to a container name in the code is here
https://github.com/pieroit/cheshire-cat/blob/main/core/cat/memory/vector_memory.py#L21

Please test everything with a fresh installation (I'll do the same in review)

Thanks

**Comment by pieroit** [comment]
solved in #177
